### PR TITLE
Finished everything including styling

### PR DESCRIPTION
**Pull request by eksheen**
Need to comment out ONE console.log()

### CODE REVIEW

File: template/quiz2.js [hunk 1]
```diff
@@ -1,39 +1,88 @@
 (function($){
+
+$.ajaxSetup({
+    async: false
+});
+	var data = $.getJSON("http://www.mattbowytz.com/simple_api.json?data=all", function (json) {
+	console.log(json);
```
Comment (eksheen) [inline template/quiz2.js]: I dont want to mess with git too much but I forgot to comment this out:(

File: template/quiz2.css [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 
 a {
 	color: #0078A0;
 }
 
-.content {
-	text-align: center
-	padding: 0 15% 20px;
+
+content {
```
Comment (matty-digital) [inline template/quiz2.css]: `.content`

File: template/quiz2.css [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
 
 .content h1 {
 	text-transform: uppercase;
 	font-weight: bold;
 	margin: 20px 0;
-
+	color: blue;
+}
 
 content form {
```
Comment (matty-digital) [inline template/quiz2.css]: `.content`

File: template/quiz2.html [hunk 4]
```diff
@@ -1,37 +1,42 @@
 <!doctype html>
 <html lang="en">
 	<head>
-		<meta charset="UTF-8"
+		<meta charset="UTF-8"</meta>
```
Comment (matty-digital) [inline template/quiz2.html]: meta tags are self closers

File: template/quiz2.html [hunk 5]
```diff
@@ -1,37 +1,42 @@
...
+						<input type="submit" value="submit" class="color1"/>
 					</form>
 				</div>
-				div class="timeout">
+				<div class="timeout">
 					<span>timeout function</span>
 				</div>
 			</div>
+			<div class ="getTitle" id="title">
+				<input type="submit" id="button" class="color" value ="Get Title"/>
```
Comment (matty-digital) [inline template/quiz2.html]: should just be a button, no form to submit to

**Pull request closed without merging**